src/components/sdk/DeviceSelection/docs/VideoFilterSelection.stories.mdx

```diff
@@ -22,14 +23,16 @@ import React from 'react';
 import {
   BackgroundBlurProvider,
   MeetingProvider,
-  VideoFilterSelection
+  BackgroundBlurCheckbox,
+  PreviewVideo
 } from 'amazon-chime-sdk-component-library-react';
 
 const App = () => {
   return (
     <BackgroundBlurProvider >
       <MeetingProvider>
-        <VideoFilterSelection />
+        <PreviewVideo />
```

Comment:
nit: Needs identation.

Reply:
PreviewVideo and Background Blur Checkbox are on the same level I think right?

Reply:
Sorry my bad.